app/src/app/home/pages/element/metric.component.html

```diff
@@ -30,7 +30,11 @@
     </div>
     <div
       class="wsl-item-topic"
-      *ngIf="meta.changeType === MetricChangeType.Same">
+      *ngIf="
+        meta.changeType === MetricChangeType.Same ||
+        meta.changeType === MetricChangeType.Faster ||
+        meta.changeType === MetricChangeType.Slower
```

Comment:
This was the real culprit. We were not reporting the actual duration when src was faster or slower.